src/content/1.7/modules/concepts/controllers/front-controllers.md

```diff
@@ -166,6 +166,17 @@ secured by redirecting it to HTTPS.
 public $ssl = true;
 ```
 
+### Ajax request
+
+When you call controller via ajax, you nedd to add ajax parameter in teh url.
+
+```php
+Context::getContext()->link->getModuleLink('cheque', 'validation', array('idPayment' => 1337, 'ajax'=>true);
+```
+
+* Without URL rewriting: `http://<shop_domain>/index.php?idPayment=1337&fc=module&module=cheque&controller=validation&id_lang=1&ajax=true`
+* With URL rewriting: `http://<shop_domain>/en/module/cheque/validation?idPayment=1337&ajax=true`
+
```

Comment:
What do you think about moving this part between `### Example of method calls` and `## Restricting access`? We could bring these examples together.

Reply:
I think it's a good idea. In ### Example of method calls is best.